test/test_interfacevalues.jl

```diff
@@ -0,0 +1,164 @@
+                    u[i] = H ⋅ xs[j]
+                    u_scal[i] = V ⋅ xs[j]
+                end
+                u_vector = reinterpret(Float64, u)
+                for i in 1:getnquadpoints(iv)
+                    @test function_gradient(iv, i, u, here = here) ≈ H
+                    @test function_symmetric_gradient(iv, i, u, here = here) ≈ 0.5(H + H')
+                    @test function_divergence(iv, i, u_scal, here = here) ≈ sum(V)
+                    @test function_divergence(iv, i, u, here = here) ≈ tr(H)
+                    @test function_gradient(iv, i, u_scal, here = here) ≈ V
+                    ndim == 3 && @test function_curl(iv, i, u, here = here) ≈ Ferrite.curl_from_gradient(H)
+
+                    @test function_value_average(iv, i, u_scal) ≈ function_value(iv, i, u_scal, here = here)
+                    @test all(function_value_jump(iv, i, u_scal) .<= 20 * eps(Float64))
+                    @test function_gradient_average(iv, i, u_scal) ≈ function_gradient(iv, i, u_scal, here = here)
+                    @test function_gradient_jump(iv, i, u_scal) <= 20 * eps(Float64)
+
+                    @test function_value_average(iv, i, u) ≈ function_value(iv, i, u, here = here)
+                    @test all(function_value_jump(iv, i, u) .<= 30 * eps(Float64))
+                    @test function_gradient_average(iv, i, u) ≈ function_gradient(iv, i, u, here = here)
+                    @test all(function_gradient_jump(iv, i, u) .<= 30 * eps(Float64))
+
+                end
+                # Test of volume
+                vol = 0.0
+                for i in 1:getnquadpoints(iv)
+                    vol += getdetJdV(iv, i; here = here)
+                end
+                
+                xs = here ? cell_coords : other_cell_coords
+                x_face = xs[[Ferrite.dirichlet_facedof_indices(scalar_interpol)[here ? face[2] : other_face[2]]...]]
+                @test vol ≈ calculate_face_area(scalar_interpol, x_face, here ? face[2] : other_face[2])
+            end
+        end
+        # Test copy
+        ivc = copy(iv)
+        @test typeof(iv) == typeof(ivc)
+        for fname in fieldnames(typeof(iv))
+            v = getfield(iv, fname)
+            v isa Ferrite.ScalarWrapper && continue
+            vc = getfield(ivc, fname)
+            if hasmethod(pointer, Tuple{typeof(v)})
+                @test pointer(v) != pointer(vc)
+            end
+            v isa FaceValues && for subfname in fieldnames(typeof(v))
+                subv = getfield(v, subfname)
+                subv isa Ferrite.ScalarWrapper && continue
+                subvc = getfield(vc, subfname)
+                if hasmethod(pointer, Tuple{typeof(subv)})
+                    @test pointer(subv) != pointer(subvc)
+                end
+                # TODO: check this
+                # @test subv == subvc #this errors for array fields and works in FaceValues test
```

Comment:
This one is the same as in `copy` test for `FaceValues` but doesn't work. * Pointers are different, values are the same*

Reply:
Turns out it's because not all faces are reinited (only interfaces) there are NaNs which breaks ==
This test is already covered in facevalues anyways.